ccmlib/node.py

```diff
@@ -1405,18 +1403,32 @@ def _set_stress_val(key, val, res):
     def stress_object(self, stress_options=None, ignore_errors=None, **kwargs):
         if ignore_errors:
             self.warning("passing `ignore_errors` to stress_object() is deprecated")
-        ret = self.stress(stress_options, **kwargs)
-        p = re.compile(r'^\s*([^:]+)\s*:\s*(\S.*)\s*$')
-        res = {}
-        start = False
-        for line in [s.strip() for s in ret.stdout.splitlines()]:
-            if start:
-                m = p.match(line)
-                if m:
-                    Node._set_stress_val(m.group(1).strip().lower(), m.group(2).strip(), res)
-            else:
-                if line == 'Results:':
-                    start = True
+        prev_locale, encoding = locale.getlocale(locale.LC_NUMERIC)
+        # Set locale to en_US to simplify printing and parsing of numbers
+        locale.setlocale(locale.LC_NUMERIC, 'en_US')
```

Comment:
if we hard code to a specific one, 

we can just replace(',',''), and remove all locale related code

setting the locale like that, might break other code we aren't even aware of, like third party code.